Web/starter_project_barbies/components/team/TeamMemberCard.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { BsFacebook, BsLinkedin, BsInstagram } from 'react-icons/bs'
+import Image from "next/image";
+import { TeamMemberCardProps } from "@/types/team/TeamMemberCardProps"
+
+const TeamMemberCard = ({
+  name,
+  jobTitle,
+  description,
+  image,
+  links,
+}: TeamMemberCardProps) => {
+  return (
+    <div className="team-card box-content font-sans w-72 mt-4 text-center shadow-md shadow-gray-200 rounded-md py-2 m-auto">
+      <div className="team-card__image container overflow-hidden my-4">
+        <Image className="rounded-full mx-auto w-32" width={200} height={200} src={image} alt={name} />
+      </div>
+
+      <div className="team-card__content m-4">
+        <h3 className="team-card__name text-2xl m-3 font-extrabold tracking-widest uppercase">
+          {name}
+        </h3>
+        <p className="team-card__job-title m-3 tracking-widest text-base uppercase">
+          {jobTitle}
+        </p>
+        <p className="team-card__description text-sm mt-5 mb-3 text-gray-400 font-light px-4 tracking-wide">
+          {description}
+        </p>
+      </div>
+
+      <hr className="h-0.5 border-t-0 bg-neutral-100 opacity-100 dark:opacity-50 block m-auto w-5/6" />
+
+      <div className="team-card__social flex-row mt-3">
+        {links && links.map((link, index) => (
+        <div key={index} className="team-card__social__icon inline-block w-1/3">
+          <a
+            className="inline-block"
+            href={link.url}
+            target="_blank"
+            rel="noreferrer"
+          >
+            {link.type === 'facebook' && <BsFacebook className="fill-gray-400 w-6 h-6" />}
```

Comment:
yeah i don't think this is nice 
we can have a key value pair that holds it like `{facebook: <BsFacebook className="fill-gray-400 w-6 h-6" />, linkedin:...}` and index that using the link.type
and for any link type that is not included, you can have a default one in there too, and refer to it with something like this
`linkIcons[link.type] || inkIcons['default']` which just uses the default one if the first one is undefined